### PR TITLE
#9807 Inconsistent usage of "Centcom" and "Centcomm" (WIP)

### PR DESCRIPTION
**Pull request by AlbertSnows**
## About the PR

We're updating all references from "centcom" to "centcomm"

## Why / Balance

Refer to #9807 

## Technical details

The change is about as simple as a find/replace all, but we still need to double check everything. There are *two* parts in this file that may be problematic. 

1) map renaming (easy)
2) url renaming (hard...maybe?)

All other changes are just file, variable, or class name changes which are fairly trivial. 

For map, we need to test that the map name change is correct and recognized. As such, loading up the referenced centcom map should be a satisfactory test. 

For the urls, I have no idea if these can be safely changed. I assume there's references to these urls in contexts I'm not aware of that rely on these URL paths. As such, a veteran developer should take a closer look at those lines before this PR ever gets merged. To be specific, the url changes I'm referring to are in the python (py) files. 

## Media

- [ TODO] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

Note: Since this is a rename, no new features are introduced. This PR will probably not need a video demonstration, it just needs to confirm that it runs and loads the touched classes and functions correctly. 

## Breaking changes

TODO

## Tests 

- [ X] It runs
- [] The centcom map loads
- [] The url's don't break anything. 

**Changelog**

:cl:
- fix: Renamed all (remaining) instances of centcomm to centcom, as fortold in the ancient texts

## Additional Notes
The commits for this PR are structured from most safe -> least safe to merge, and thus can be unwound as needed.

### CODE REVIEW

**Comment by deltanedas** [comment]
this was already done no

**Comment by CatMagic1** [comment]
it's central command not central comand 

(nothing but fax and logic here folks)

**Comment by Emisse** [comment]
all of our internal database stuff uses centcomm, centcom is the wrong form of it

**Comment by deltanedas** [comment]
**cent**ral **comm**and
centcomm

**Comment by LankLTE** [comment]
> **cent**ral **comm**and
> centcomm

**Cent**ral **Com**and
CentCom

I agree it’s CentComm just saying that argument doesn’t really hold up to why

**Comment by AlbertSnows** [comment]
Peptide suggested it's CentCom
https://github.com/space-wizards/space-station-14/issues/9807#issuecomment-1503224655
Will give this another look when I have time, and either fix or close accordingly.

**Comment by deltanedas** [comment]
> I agree it’s CentComm just saying that argument doesn’t really hold up to why

CentComm is 4/7 letters from each word
CentCom is 4/7 and 3/7 which is evil

**Comment by QuietlyWhisper** [comment]
If its supposed to be centcomm and not centcom someone should probably go revert the pr that changed a lot of things to "centcom". Also probably important to go clearify for certain that this is the correct usage on the original issue

**Comment by mirrorcult** [comment]
image

maintainer decision

**Comment by deltanedas** [comment]
another W for centcomm enjoyers

File: Content.Server/Shuttles/Systems/EmergencyShuttleSystem.cs [hunk 1]
```diff
@@ -72,8 +72,8 @@ public override void Initialize()
         _configManager.OnValueChanged(CCVars.EmergencyShuttleEnabled, SetEmergencyShuttleEnabled);
         SubscribeLocalEvent<RoundStartingEvent>(OnRoundStart);
         SubscribeLocalEvent<StationEmergencyShuttleComponent, ComponentStartup>(OnStationStartup);
-        SubscribeLocalEvent<StationCentcommComponent, ComponentShutdown>(OnCentcommShutdown);
-        SubscribeLocalEvent<StationCentcommComponent, ComponentInit>(OnCentcommInit);
+        SubscribeLocalEvent<StationCentcommComponent, ComponentShutdown>(OnCentcomShutdown);
```
Comment (AlbertSnows) [inline Content.Server/Shuttles/Systems/EmergencyShuttleSystem.cs:75]: fix

**Comment by AlbertSnows** [comment]
Current status: first draft of switching to centcomm. It has NOT been tested.
Next steps: 
- address failing tests (probably typo or missed one)
- quindrillion check changes (and consider maybe separating releases?)
- run the game and try to test as many different changed lines as possible

**Comment by ike709** [comment]
CentCom is enforced in CI as the correct spelling across several SS13 servers. Democracy is objectively wrong.

**Comment by QuietlyWhisper** [comment]
> CentCom is enforced in CI as the correct spelling across several SS13 servers. Democracy is objectively wrong.

Theres some backend reasoning for using centcomm for consistancy, while I do like centcom better it just makes more sense

**Comment by TGODiamond** [comment]
I like to call central commanders "CentCommers", just for fun. "Those damn CentCommers, docking the emergency shuttle to cargo!"

**Comment by notquitehadouken** [comment]
centco

**Comment by QuietlyWhisper** [comment]
Sorry for merge conflict but I made sure to use CentComm in my files.

**Comment by AJCM-git** [comment]
@AlbertSnows You coming back to this?

**Comment by AlbertSnows** [comment]
> @AlbertSnows You coming back to this?

https://github.com/space-wizards/space-station-14/pull/21108#issuecomment-1772062146

**Comment by AJCM-git** [comment]
Aight, when you do have time feel free to reopen. If someone wants to pick it up also open a new pr.

**Pull request closed without merging**